src/overlays/actors/ovl_En_Bal/z_en_bal.c

```diff
@@ -15,7 +29,79 @@ void EnBal_Destroy(Actor* thisx, PlayState* play);
 void EnBal_Update(Actor* thisx, PlayState* play);
 void EnBal_Draw(Actor* thisx, PlayState* play);
 
-#if 0
+typedef enum EnBalEyeTexture {
+    /* 0 */ EN_BAL_EYETEX_OPEN,
+    /* 1 */ EN_BAL_EYETEX_CLOSED
+} EnBalEyeTexture;
+
+typedef enum EnBalAnimation {
+    /*  0 */ TINGLE_ANIM_FLOAT_DRAW,
+    /*  1 */ TINGLE_ANIM_FALL_LOOP,
+    /*  2 */ TINGLE_ANIM_FALL_ONCE,
+    /*  3 */ TINGLE_ANIM_LAND,
+    /*  4 */ TINGLE_ANIM_TALK_TWIST,
+    /*  5 */ TINGLE_ANIM_TALK_IDLE,
+    /*  6 */ TINGLE_ANIM_MAGIC_FULL,
+    /*  7 */ TINGLE_ANIM_HAPPY_DANCE_LOOP,
+    /*  8 */ TINGLE_ANIM_HAPPY_DANCE_ONCE,
+    /*  9 */ TINGLE_ANIM_MAGIC_SHORT,
+    /* 10 */ TINGLE_ANIM_WAIT,
+    /* 11 */ TINGLE_ANIM_SPIN,
+    /* 12 */ TINGLE_ANIM_HIDE_FACE,
+    /* 13 */ TINGLE_ANIM_CONFETTI,
+    /* 14 */ TINGLE_ANIM_MAX
+} EnBalAnimation;
+
+typedef enum EnBalBuyMapChoice {
+    /* 0 */ EN_BAL_MAPCHOICE_PROXIMAL,
+    /* 1 */ EN_BAL_MAPCHOICE_DISTAL,
+    /* 2 */ EN_BAL_MAPCHOICE_CANCEL
+} EnBalBuyMapChoice;
+
+typedef enum EnBalWatchTarget {
+    /* 0 */ EN_BAL_WATCHTRG_NONE,
```

Comment:
```suggestion
typedef enum EnBalWatchTarget {
    /* 0 */ TINGLE_WATCH_TARGET_NONE,
```
I think you can afford the full word here, `TRG` is an abbreviation I would never guess. Also, `EN_BAL` -> `TINGLE` for all these enums?